Assets/MixedRealityToolkit/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -46,7 +47,7 @@ public class MixedRealityProjectConfiguratorWindow : EditorWindow
         private readonly GUIContent LaterButtonContent = new GUIContent("Later", "Do not show this pop-up notification until next session");
         private readonly GUIContent IgnoreButtonContent = new GUIContent("Ignore", "Modify this preference under Edit > Project Settings > MRTK");
 
-        private bool showConfigurations = false;
+        private bool showConfigurations = true;
```

Comment:
Was this in intentional part of this change? i.e. this doesn't seem to be part of handling legacy APIs.